src/components/OSCALMetadata.js

```diff
@@ -38,17 +53,174 @@ const OSCALMetadataPartiesCard = styled(Paper)(({ theme }) => ({
   backgroundColor: theme.palette.background.paper,
   position: "relative",
   overflow: "auto",
-  maxHeight: "12em",
+  maxHeight: "18em",
 }));
 
 // Returns a string with a locality-sensitive representation of this date
 const formatDate = (isoDate) => new Date(isoDate).toLocaleString();
 
+const TELEPHONE_ICON_MAPPING = {
+  home: <HomeIcon />,
+  mobile: <SmartphoneIcon />,
+  office: <BusinessIcon />,
+};
+
+const ADDRESS_ICON_MAPPING = {
+  home: <HomeIcon />,
+  work: <BusinessIcon />,
+};
+
+const UNKNOWN_TYPE_ICON = <HelpOutlineIcon />;
+
+function getPhoneIcon(contactType) {
+  return TELEPHONE_ICON_MAPPING[contactType] ?? UNKNOWN_TYPE_ICON;
+}
+
+function getAddressIcon(contactType) {
+  return ADDRESS_ICON_MAPPING[contactType] ?? UNKNOWN_TYPE_ICON;
+}
+
+export function OSCALMetadataPartyAddress(props) {
+  const { address } = props;
+  return (
+    <>
+      {getAddressIcon(address.type)}
+      <address>
+        <Typography>
+          {`${address["addr-lines"]?.join(", ")} ${address.city} ${
+            address["postal-code"]
+          } ${address.country}`}
+        </Typography>
+      </address>
+    </>
+  );
+}
+
+export function OSCALMetadataPartyEmail(props) {
+  return (
+    <Typography>
+      <Link href={`mailto:${props.email}`}>{props.email}</Link>
+    </Typography>
+  );
+}
+
+export function OSCALMetadataPartyTelephone(props) {
+  const { telephone } = props;
+  return (
+    <Typography>
+      {getPhoneIcon(telephone.type)}
+      <Link href={`tel:${telephone.number}`}>{telephone.number}</Link>
+    </Typography>
+  );
+}
```

Comment:
I think these would make really good Storybook items. Please add these there. Actually, almost everything here probably would be a good addition.